probe-rs/src/probe/daplink/mod.rs

```diff
@@ -357,6 +358,36 @@ impl DebugProbe for DAPLink {
         Ok(())
     }
 
+    fn target_reset_assert(&mut self) -> Result<(), DebugProbeError> {
+        let mut mask = Pins(0);
+        mask.set_nreset(true);
+        mask.set_ntrst(true);
+        let request = SWJPinsRequest {
+            output: mask,
+            select: Pins(0x00),
```

Comment:
I think these are the wrong way around; `output` is the value to output (we want 0 to assert reset) and `select` is the pins to change.

```suggestion
            output: Pins(0),
            select: mask,
```